src/vs/workbench/contrib/chat/electron-sandbox/tools/fetchPageTool.ts

```diff
@@ -138,11 +138,21 @@ export class FetchWebPageTool implements IToolImpl {
 			let confirmationTitle: string;
 			let confirmationMessage: string | MarkdownString;
 			if (urlsNeedingConfirmation.length === 1) {
-				confirmationTitle = localize('fetchWebPage.confirmationTitle.singular', 'Fetch untrusted web page?');
-				confirmationMessage = urlsNeedingConfirmation[0].toString();
+				confirmationTitle = localize('fetchWebPage.confirmationTitle.singular', 'Fetch web page?');
+				confirmationMessage = new MarkdownString(
+					localize('fetchWebPage.confirmationMessage.singular', 
+						'{0}\n\n$(info) Web content may contain malicious code or attempt prompt injection attacks.', 
```

Comment:
Dont include the `\n\n$(info)` in the string that is going to be translated and  instead concatenate it to the end of the argument that will be injected at runtime